examples/svelte/.gitignore

```diff
@@ -0,0 +1,24 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
+lerna-debug.log*
+
+node_modules
+dist
+dist-ssr
+*.local
+
+# Editor directories and files
+.vscode/*
+!.vscode/extensions.json
+.idea
```

Comment:
Seems like it should NOT be ignored https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems (apart from user files)

Reply:
That was generated by vite, happy to remove it across all projects if needed , but seems to be related to developers environment configuration rather than project. I.e. if we both use intellij editors with custom setup - we would have conflicte?